### PR TITLE
improve coverage for auth

### PR DESCRIPTION
**Pull request by apapirovski**
Add test case to cover the error state of `getCredentials`. Also changed `auth` to use `os.EOL`.

Fixes: #36

### CODE REVIEW

**Comment by apapirovski** [comment]
@joyeecheung is there any way to run `appveyor` for specific PR? Curious if this does anything to those failing tests.

**Comment by joyeecheung** [comment]
@apapirovski Couldn't find any option for that in the console...although I think you can enable it in your own fork to see how it goes?

**Comment by apapirovski** [comment]
(I think I should be able to get these tests fixed up. Will post when done.)

File: test/unit/auth.test.js [hunk 1]
```diff
@@ -112,6 +112,7 @@ function runAuthScript(ncurc = undefined, expect = [], error = '', fixture = 'ru
     });
 
     function onLine(line) {
+      console.log(line);
```
Comment (cPhost) [inline test/unit/auth.test.js]: I think you forgot to remove this.

**Comment by apapirovski** [comment]
Got appveyor all fixed up. Once this lands, we can re-enable it for PRs. 😄

**Comment by joyeecheung** [comment]
Awesome! It's all green now, thanks for working on this@ @apapirovski https://ci.appveyor.com/project/joyeecheung/node-core-utils/build/59.$%7Bpatch%7D

**Pull request merged**